library/components/TDropdown.vue

```diff
@@ -13,12 +11,11 @@
         </div>
 		<slot name="outside"></slot>
 		<div
-			v-show="isPopupOpen"
 			ref="popup"
-			class="absolute mt-1 bg-white rounded-lg shadow-md base-dropdown-menu w-max min-w-"
-			:class="alignClass"
+			class="absolute mt-1 bg-white rounded-lg shadow-md base-dropdown-menu w-max"
+			:class="[alignClass, isPopupOpen ? 'h-max': 'h-[0px] opacity-0 overflow-y-hidden']"
```

Comment:
I think that this dynamic part is not working as expected, it's not producing any issues but it's not changing neither:
`isPopupOpen ? 'h-max': 'h-[0px] opacity-0 overflow-y-hidden']`


![2022-08-10 10 58 27](https://user-images.githubusercontent.com/1915140/183860391-7f4f7e2a-72da-4bbe-8015-0756ce1bfb78.gif)

The one that is actually changing is the TExpandable.vue one

Reply:
Yes this is just setting height of expandable section, elements still stay in DOM just height gets reduced and max on click.

`height: 0px` gets changed to `height: max-content`

Am I misunderstanding you question?

Reply:
yes, you have modified and add this behaviour to two components TExpandable and TDropdown. The one changing in the DOM is the TExpandable while the TDropdown is always hardcoded as `h-max` and it's not reacting to the `isPopupOpen` data change.
On the GIF I shared, the DOM node that is highlighted, correspond to the TDropdown, is not changing. The ones changing are the one from the TExapandable.

Reply:
Ahh I see what you mean, yes the t-dropdown only changes when you first time open the filters dropdown, that is a wrapper container.

While t-expandable are child component inside the t-dropdown.

https://user-images.githubusercontent.com/43262405/183873267-6fa1d440-c504-4871-92ea-bb3f1b7cc25e.mp4